.github/workflows/flowkit-release.yml

```diff
@@ -0,0 +1,18 @@
+name: Flowkit Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  flowkit-release:
+    if: "!github.event.release.prerelease"
```

Comment:
Why do we not want it for prereleases?

Reply:
@bjartek I think we'd want this from the publish release here
https://github.com/onflow/flow-cli/blob/master/.github/workflows/publish-release.yml#L7-L8

I figured we just do it as similar as that workflow is doing, as previously i opted in adding this step as part of `publish release`.

Reply:
Yeah then it makes sende.

Reply:
For some reason i was thinking prerelease tag. But this is github release bot git tag related